packages/fiori/src/fcl-utils/FCLLayout.ts

```diff
@@ -1,22 +1,37 @@
-const getLayoutsByMedia = () => {
+import { MEDIA } from "../FlexibleColumnLayout.js";
+
+type LayoutConfiguration = {
+	[key in MEDIA]: {
+		[key: string]: {
+			layout: Array<string>;
```

Comment:
@ilhan007  could you check that too. Previously it was accepting```100%```, ```100px``` and ```0```. This may be breaking change?